CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5548](https://github.com/thanos-io/thanos/pull/5548) Query: Added experimental support for load balancing across multiple Store endpoints.
 - [#6148](https://github.com/thanos-io/thanos/pull/6148) Query-frontend: add traceID to slow query detected log line
 - [#6153](https://github.com/thanos-io/thanos/pull/6153) Query-frontend: add remote_user (from http basic auth) and remote_addr to slow query detected log line
+- [#6174](https://github.com/thanos-io/thanos/pull/6174) Tools bucket replicate: Blocks are replicated concurrently.
```

Comment:
I don't think this is a helpful changelog tbh. Can you please also document the user-facing changes? Like we added a new flag

Reply:
And please update the changelog. This section has been released already, please add it to the unreleased section.

Reply:
Made the suggested changes